app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AnswersController < ApplicationController
+  before_action :set_answer, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!
+  authorize_resource
+  # GET /answers/1/edit
+  def edit
+  end
+  
+  # POST /answers
+  # POST /answers.json
+  def create
```

Comment:
these should be nested routes of question.
for reference google "Restful nested routes"